src/test/blk_recovery/TEST0

```diff
@@ -45,6 +45,8 @@ require_test_type medium
 # doesn't make sense to run in local directory
 require_fs_type pmem non-pmem
 require_build_type nondebug static-nondebug
+configure_valgrind helgrind force-disable
```

Comment:
Please add comments before each force-disable WHY you are doing that. I suspect I know why you do it for recovery tests (but still, this information should be here) but I don't know why it's not a problem for memcheck and I have no idea why you disable it for blk_rw_mt.